### PR TITLE
Pf2 implementation

### PR DESCRIPTION
**Pull request by MrBones1102**
Adding parafac2 analysis of time data in the CA dataset, and of patients across diseases

### CODE REVIEW

File: mrsa_ca_rna/import_data.py [hunk 1]
```diff
@@ -272,7 +272,7 @@ def import_ca_val_rna():
     return ca_val_rna
 
 
-def extract_time_data():
+def extract_time_data(scaled: tuple = (True, 0)):
```
Comment (aarmey) [inline mrsa_ca_rna/import_data.py:275]: This is an unusual way to pass in this information. I would recommend splitting this into two arguments, `scale` boolean and `scale_value`.

**Review by aarmey — CHANGES_REQUESTED** [review]
Looks good! One suggestion here. Also, please resolve the merge conflict. Have you taken a look at why the build is failing?

File: mrsa_ca_rna/factorization.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    # convert the xarray dataset to a numpy list
+    data_list = []
+    for data_var in data.data_vars:
+        data_list.append(data[data_var].values)
+
+    # data_np = [data["MRSA"].values, data["Candidemia"].values]
+
+    # perform the factorization
+    (weights, factors, projection_matrices), rec_errors = tl.decomposition.parafac2(
+        data_list, rank=rank, n_iter_max=2000, verbose=True, return_errors=True
```
Comment (aarmey) [inline mrsa_ca_rna/factorization.py:109]: @MrBones1102 use `init="svd"`, `svd="randomized_svd"`, and `normalize_factors=True`. This should provide a faster and more consistent result.

**Comment by MrBones1102** [comment]
> Looks good! One suggestion here. Also, please resolve the merge conflict. Have you taken a look at why the build is failing?

Yes, I just went through yesterday and some of my early figure files need refactoring to work with the AnnData methods I'm using instead of the multiindex pandas.

I plan to have everything fixed up by the end of the day today.

**Pull request merged**